src/utils/FormElementMixin.js

```diff
@@ -85,7 +86,9 @@ export default {
         onBlur($event) {
             this.isFocused = false
             this.$emit('blur', $event)
-            this.checkHtml5Validity()
+            if (this.useHtml5Validation || config.defaultUseHtml5Validation) {
```

Comment:
Remove config value check.
Are you sure is it enough ? Isn't it better add new prop check into checkHtml5Validity method?